rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/objectstore/TestBase.java

```diff
@@ -50,6 +50,16 @@ public Set<Class<?>> getClasses() {
         }
     }
 
+    @ApplicationPath("/")
+    public static class LRACoordinator extends Application {
```

Comment:
How about extending `JaxRsApplication` - would we get the OpenAPI deployed so it could be tested that there is no /lra-coordinator/lra-coordinator references in the yaml?